README-development.md

```diff
@@ -1,3 +1,39 @@
+# Operating System
+
+## Prerequisites
+
+- git
+- coreos-assembler
+- rpmdistro-gitoverlay
+- ostree
+- rpm-ostree 
+- imagefactory-plugins-TinMan
+- qemu-img
```

Comment:
Hmm, might be worth inlining these requires into the building steps. That way if one just wants to know how to compose an OSTree, they don't need to install e.g. imgfac (which pulls in libvirt + systemd).

Reply:
Offline suggestion: link the exotic tools to their repo (or, if they have it, to their "how to install" doc).

Reply:
Most of this is `yum -y install`, with a few exceptions, those we should call out.